commands/command.md

```diff
@@ -57,34 +84,36 @@ Command arity _includes_ counting the command name itself.
 
 Examples:
 
-  - `GET` arity is 2 since the command only accepts one
-argument and always has the format `GET _key_`.
-  - `MGET` arity is -2 since the command accepts at a minimum
-one argument, but up to an unlimited number: `MGET _key1_ [key2] [key3] ...`.
+  - `GET` arity is 2 since the command only accepts one argument and always has the format `GET _key_`.
+  - `MGET` arity is -2 since the command accepts at a minimum one argument, but up to an unlimited number: `MGET _key1_ [key2] [key3] ...`.
 
 Also note with `MGET`, the -1 value for "last key position" means the list
 of keys may have unlimited length.
 
-### Flags
-Command flags is @array-reply containing one or more status replies:
-
-  - *write* - command may result in modifications
-  - *readonly* - command will never modify keys
-  - *denyoom* - reject command if currently out of memory
-  - *admin* - server admin command
-  - *pubsub* - pubsub-related command
-  - *noscript* - deny this command from scripts
-  - *random* - command has random results, dangerous for scripts
-  - *sort\_for\_script* - if called from script, sort output
-  - *loading* - allow command while database is loading
-  - *stale* - allow command while replica has stale data
-  - *skip_monitor* - do not show this command in MONITOR
-  - *asking* - cluster related - accept even if importing
-  - *fast* - command operates in constant or log(N) time.  Used for latency monitoring.
-  - *movablekeys* - keys have no pre-determined position.  You must discover keys yourself.
-
+## flags
 
-### Movable Keys
+Command flags is @array-reply containing one or more status replies:
+  - `write`: command may result in modifications
+  - `readonly`: command will never modify keys
+  - `denyoom`: reject command if currently out of memory
+  - `admin`: server admin command
+  - `pubsub`: pubsub-related command
+  - `noscript`: deny this command from scripts
+  - `random`: command has random results, dangerous for scripts
+  - `sort_for_script`: if called from script, sort output (Deprecated since 7.0.0)
+  - `loading`: allow command while database is loading
```

Comment:
`no_async_loading` missing

Reply:
what about CMD_PROTCTED? do we want it here?
also, it's not documented next to the other CMD_*, just about the definition of `redisCommand`

Reply:
IIRC we decided not to expose CMD_PROTECTED in COMMAND command.
but we did expose `no_async_loading`

Reply:
we do expose "protected" (see addReplyFlagsForCommand)

Reply:
i must be drunk, or have been.
seems like i did that in https://github.com/redis/redis/pull/10015
i'm leaning towards removing it rather than documenting it.
WDYT?

Reply:
If it ain't reported, it doesn't need to be documented

Reply:
ok, let's hide it from COMMAND (i'll add it to my TODO list)